pkgs/by-name/fr/frr-exporter/package.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "frr-exporter";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "tynany";
+    repo = "frr_exporter";
+    rev = "v${version}";
+    hash = "sha256-sjC6ee7VMbr5zlahAA/i4wrZIu8CBUF+LYaq6uEvukk=";
+  };
+
+  vendorHash = "sha256-xckGN/FKN2bwFPll6IRDBOGGUm2JTzoZDwKC6ZkoD/Y=";
+
+  ldflags = [
+    "-X github.com/prometheus/common/version.Version=${version}"
+    "-X github.com/prometheus/common/version.Revision=${src.rev}"
+    "-X github.com/prometheus/common/version.Branch=unknown"
+  ];
+
+  meta = with lib; {
+    description = ''
+      Prometheus exporter for FRR version 3.0+ that collects metrics from the
+      FRR Unix sockets and exposes them via HTTP, ready for collecting by
+      Prometheus.
+    '';
```

```suggestion
    longDescription = ''
      Prometheus exporter for FRR version 3.0+ that collects metrics from the
      FRR Unix sockets and exposes them via HTTP, ready for collecting by
      Prometheus.
    '';
```

> `meta.description` must:
    Be short, just one sentence.
    Be capitalized.
    Not start with the package name.
    More generally, it should not refer to the package name.
    Not end with a period (or any punctuation for that matter).
    Provide factual information.
    Avoid subjective language.

ref: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

You should only put one sentence in `meta.description` and not end it with a period. You can use `meta.longDescription` instead. You must also set `meta.description` to follow these rules.